### PR TITLE
Feature/issue 128/add slideshow

### PR DESCRIPTION
**Pull request by catthiba**
Happy new year Alex!

### CODE REVIEW

File: internal/static/internal/css/frontpage.css [hunk 1]
```diff
@@ -38,15 +38,15 @@
...
 .frontpage__image {
     width: 650px;
-    margin: 0 10px;
+    margin: 0px;
     height: 380px;
     background-image: url("https://cdn.pixabay.com/photo/2017/06/16/07/26/under-construction-2408061_1280.png");
     background-repeat: no-repeat;
     background-position: 50% 50%;
-}
+}*/
```
Comment (alexaor) [inline internal/static/internal/css/frontpage.css]: If this doesn't do anything could probably just get rid of it

File: internal/static/internal/css/frontpage.css [hunk 2]
```diff
@@ -119,3 +119,61 @@
...
+    70% { left: -200%; }
+    75% { left: -300%; }
+    95% { left: -300%; }
+    100% {left: -400%; }
+    } 
+       
+
+
+
+body { margin: 0; }
```
Comment (alexaor) [inline internal/static/internal/css/frontpage.css]: We already have a root tag dealing with this would just be clutter. See common/static/css/main if you're curious.

File: internal/static/internal/css/frontpage.css [hunk 3]
```diff
@@ -119,3 +119,61 @@
...
+    box-shadow: 0 0 20px rgba(0, 0, 0, 0.4);
+  }
+  
+  #slideshow > div {
+    position: absolute;
+    top: 10px;
+    left: 10px;
+    right: 10px;
+    bottom: 10px;
+  }*/
```
Comment (alexaor) [inline internal/static/internal/css/frontpage.css]: Same as commented out css above

File: internal/static/internal/css/frontpage.css [hunk 4]
```diff
@@ -119,3 +119,61 @@
     margin-bottom: 0.4em;
     box-sizing: border-box;
 }
+
+/* kode for slideshow */
+@keyframes slidy {
```
Comment (alexaor) [inline internal/static/internal/css/frontpage.css]: comment could be replaced with self-documenting naming, meanig that the slidy -> slideshow or something like that making it a bit more obvious (although slidy is a catchy name).

File: internal/static/internal/css/header.css [hunk 5]
```diff
@@ -116,7 +116,7 @@
 
 .dropdown-content a:hover {
     background-color: #103242;
-    color: #BBD8E4;
+    color: rgb(228, 203, 187);
```
Comment (alexaor) [inline internal/static/internal/css/header.css]: Why the change here?

File: internal/views.py [hunk 6]
```diff
@@ -17,11 +18,13 @@ def index(request):
...
         key=lambda instance: instance.created
     )
-    ctx = {
+    ctx = { 
         'last_summaries': last_summaries,
         'last_quotes': last_quotes,
-        'next_shifts': request.user.shift_set.filter(slot__group__meet_time__gte=timezone.now())[:2],
-    }
+        'next_shifts': request.user.shift_set.filter(slot__group__meet_time__gte=timezone.now())[:2], 
+        'slideshow': SlideshowImage.objects.filter(start_datetime__lte=timezone.now()).filter(end_datetime__gte=timezone.now()),
```
Comment (alexaor) [inline internal/views.py]: `filter` accepts multiple keyword arguments, meaning that it should be possible to write something like this.

```suggestion
        'slideshow': SlideshowImage.objects.filter(start_datetime__lte=timezone.now(), end_datetime__gte=timezone.now()), 
```

**Review by alexaor — CHANGES_REQUESTED** [review]
Great work! Got some minor comments about commented out code and simplifying some filtering. Should be mergeable once that is in order 😊

File: internal/static/internal/css/header.css (hunk 5 above)
Comment (catthiba) [reply internal/static/internal/css/header.css]: Screenshot from 2021-01-11 13-25-18

I can't recall why

**Review by alexaor — APPROVED** [review]
LGTM
